client/landing/subscriptions/tab-views/sites.tsx

```diff
@@ -1,23 +1,14 @@
+import { SubscriptionManager } from '@automattic/data-stores';
 import SiteList from '../site-list/site-list';
+import type { SiteSubscription } from '@automattic/data-stores/src/reader/types';
 
-// Mock meanwhile we have the data endpoints ready
-const useSites = () => {
-	const sites = [];
-	for ( let i = 0; i < 10; i++ ) {
-		sites.push( {
-			id: i,
-			name: `Site ${ i }`,
-			icon: `https://www.gravatar.com/avatar/${ i }?s=96&d=mm`,
-			url: `https://site${ i }.wordpress.com`,
-			date: new Date(),
-			emailFrequency: [ 'Daily', 'Weekly', 'Immediately' ][ Math.floor( Math.random() * 3 ) ],
-		} );
-	}
-	return sites;
+type useSiteSubscriptionsQueryResponse = {
+	data: SiteSubscription[];
 };
 
 export default function SitesView() {
-	const sites = useSites();
+	const { data: sites } =
+		SubscriptionManager.useSiteSubscriptionsQuery() as useSiteSubscriptionsQueryResponse;
```

Comment:
Why do we need the `as useSiteSubscriptionsQueryResponse`? 🤔

Reply:
Great catch! Thanks, Maciej! I have fixed that in https://github.com/Automattic/wp-calypso/pull/75115/commits/854fc81bde6b42567debd765efdbe18381a9a53c.